platform/pulp/platform/models/repository.py

```diff
@@ -214,7 +214,6 @@ class Importer(Plugin):
     scratchpad = fields.GenericRelation(Scratchpad)
 
     class Meta:
-        abstract = True
         unique_together = ('name', 'repository')
```

Comment:
So the unfortunate deal with this is that if ``Importer`` is not abstract, a table gets created for and now ``name`` is in the ``Importer`` table and ``repository`` is in the ``RepositoryImporter`` table. Unfortunately it's not possible to use ``unique_together`` with multi-table inheritance.

We had a conversation earlier today about all this, and I'm not sure whether we want to defer dealing with the Importer/RepositoryImporter and Distributor/RepositoryDistributor thing until later, or if we want to address it now.